tests/test_devices.py

```diff
@@ -255,6 +255,152 @@ def test_device_inventory_update_success(client, mocker):
     assert device._inventory == inventory
 
 
+class test_inputs:
```

Comment:
Classes must always use CapWords convention (https://peps.python.org/pep-0008/#class-names). Snakecase (like `test_inputs` is fine for functions, but not for classes. In this case it should be `TestInputs` or `TestInputsView` (as it is a view over `_inventory`).